### PR TITLE
Replace http://example.{com,org}/ with https://example.{com,org}/

### PR DESCRIPTION
**Pull request by mpcabd**
To be consistent with gohugoio/hugo, @bep suggests changing all
instances of http://(www.)?example.{com,org} with
https://(www.)?example.{com,org}

See issue gohugoio/hugo#3728 with pull request gohugoio/hugo#3739

### CODE REVIEW

**Comment by digitalcraftsman** [comment]
@bep are the changes now as intended?

File: content/content-management/urls.md [hunk 1]
```diff
@@ -200,15 +200,15 @@ See [Content Organization][contentorg] for more details on paths.
 .
 └── content
     └── about
-    |   └── _index.md  // <- http://example.com/about/
+    |   └── _index.md  // <- httpss://example.com/about/
```
Comment (digitalcraftsman) [inline content/content-management/urls.md]: Is `httpss` the new **super**-secure hypertext transfer protocol? :wink:

Comment (mpcabd) [inline content/content-management/urls.md]: Just fixed it in a122578

Sorry 😅

Comment (digitalcraftsman) [inline content/content-management/urls.md]: Thanks

**Pull request merged**